make-based/app-elfloader/do.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+
+app_basename="elfloader"
+app_libs="lwip zydis libelf"
+use_networking=1
+use_9p_rootfs=1
+rootfs_9p="fs0"
+use_kvm=1
+
+# https://stackoverflow.com/a/246128/4804196
+SOURCE=${BASH_SOURCE[0]}
+while [ -L "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+    DIR=$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )
+    SOURCE=$(readlink "$SOURCE")
+    [[ $SOURCE != /* ]] && SOURCE=$DIR/$SOURCE # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+SCRIPT_DIR=$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )
+
+source "$SCRIPT_DIR"/../include/base
+
+app="$top"/apps/app-"$app_basename"
+
+source "$SCRIPT_DIR"/../include/common_functions
+
+setup_elfloader()
+{
+    git remote rm csvancea > /dev/null 2>&1
+    git remote add csvancea https://github.com/csvancea/app-elfloader/
+    git fetch csvancea
+    git checkout staging
+    git branch -D csvancea/fix/argv-parser > /dev/null 2>&1
+    git branch -D csvancea/fix/unaligned-stack > /dev/null 2>&1
+    git branch csvancea/fix/argv-parser csvancea/csvancea/fix/argv-parser
+    git branch csvancea/fix/unaligned-stack csvancea/csvancea/fix/unaligned-stack
+    git branch -D setup > /dev/null 2>&1
+    git checkout -b setup
+    git rebase csvancea/fix/argv-parser
+    git rebase csvancea/fix/unaligned-stack
+
+    if test ! -d ../run-app-elfloader; then
+        git clone https://github.com/unikraft/run-app-elfloader ../run-app-elfloader
+    fi
+
+    if test ! -d ../static-pie-apps; then
+        git clone https://github.com/unikraft/static-pie-apps ../static-pie-apps
+    fi
```

Comment:
I think the [`dynamic-apps` repository](https://github.com/unikraft/dynamic-apps) should also be listed here.
Running anything not static gives `failed to open '../dynamic-apps/[...]': No such file or directory`

Reply:
Indeed, will fix.